--- build/tools/make_standalone_toolchain.py	2019-01-14 18:23:12.000000000 +0100
+++ build/tools/make_standalone_toolchain_changed.py	2019-01-26 12:55:10.510801480 +0100
@@ -127,7 +127,7 @@
 
     triple = get_triple(arch)
     target = make_clang_target(triple, api)
-    flags = '-target {} -stdlib=libc++'.format(target)
+    flags = '-target {}'.format(target)
 
     # We only need mstackrealign to fix issues on 32-bit x86 pre-24. After 24,
     # this consumes an extra register unnecessarily, which can cause issues for
@@ -161,7 +161,7 @@
         clangpp.write(textwrap.dedent("""\
             #!/bin/bash
             if [ "$1" != "-cc1" ]; then
-                `dirname $0`/clang{version}++ {flags} "$@"
+                `dirname $0`/clang{version}++ -stdlib=libc++ {flags} "$@"
             else
                 # target/triple already spelled out.
                 `dirname $0`/clang{version}++ "$@"
